assets/src/scss/editorOverrides.scss

```diff
@@ -399,57 +399,6 @@ input.describe[type=text][data-setting=caption] {
   }
 }
 
-.posts-list,
-.actions-list {
-  .carousel-controls,
-  .wp-block-query-pagination {
-    display: none !important;
-  }
-}
-
-// Posts List Block Override for the List layout
-.posts-list.is-custom-layout-list {
-  .wp-block-post-template.is-layout-flow li {
-    height: 240px;
-    margin-bottom: 12px;
-  }
-
-  .wp-block-post-featured-image {
-    flex-basis: 30%;
-  }
-
-  .wp-block-group.is-layout-flow {
-    flex-basis: 65%;
-  }
-
-  .wp-block-post-title {
-    margin-top: -25px !important;
-  }
-
-  .wp-block-post-excerpt {
-    margin-top: -24px;
-
-    &__more-text {
-      display: none;
-    }
-  }
-
-  .is-layout-flow > .is-layout-flow {
-    margin-top: -36px;
-  }
-
-  @media (max-width: 991px) {
-    .wp-block-post-title,
-    .wp-block-post-excerpt {
-      margin-top: -12px;
-    }
```

Comment:
Have we moved these classes to the new SCSS files?

Reply:
No, these are no longer needed if we remove the custom margins from the plugin